README.md

```diff
@@ -155,13 +153,27 @@ See the [open issues](https://github.com/esa/torchquad/issues) for a list of pro
 
 The project is open to community contributions. Feel free to open an [issue](https://github.com/esa/torchquad/issues) or write us an email if you would like to discuss a problem or idea first.
 
+To get the most up-to-date code, you can follow this quick guide for installing *torchquad* from source:
+
+1. Get [miniconda](https://docs.conda.io/en/latest/miniconda.html) or similar
+2. Clone the repo
+   ```sh
+   git clone https://github.com/esa/torchquad.git
+   ```
+3. Setup the environment. This will create a conda environment called `torchquad`
+   ```sh
+   conda env create -f environment.yml
+   ```
+
+Once the installation is done, then
```

Comment:
This doesn't make sense. You fork first, then install, then proceed with step 2.

Reply:
👍